Mathlib/Data/List/Sort.lean

```diff
@@ -70,6 +70,23 @@ theorem Sorted.tail {r : α → α → Prop} {l : List α} (h : Sorted r l) : So
 theorem rel_of_sorted_cons {a : α} {l : List α} : Sorted r (a :: l) → ∀ b ∈ l, r a b :=
   rel_of_pairwise_cons
 
+theorem Sorted.cons {r : α → α → Prop} [IsTrans α r] {l : List α} {a b : α}
+    (h : r b a) (ha : Sorted r (a :: l)) : Sorted r (b :: a :: l) := by
+  refine List.pairwise_cons.2 ⟨?_, ha⟩
+  rintro c (_ | _)
+  · exact h
+  · apply trans h
+    apply rel_of_sorted_cons ha
+    assumption
+
+theorem Sorted.cons_iff {r : α → α → Prop} [IsTrans α r] {l : List α} {a b : α} :
```

Comment:
Arguably this should be called `sorted_cons` to match `pairwise_cons`

Reply:
I think this is more in line with lean 4, though

Reply:
Then we should rename `pairwise_cons` too

Reply:
Do you want to do that additional rename here? I don't.

Reply:
Is that an uncontroversial rename? I can do it in a separate PR.

Reply:
> I think this is more in line with lean 4, though

Revisiting, I don't know if I agree with this claim. I'll ping @YaelDillies for another opinion.

Reply:
I agree. This should be `sorted_cons`